clouddrift/adapters/gdp1h.py

```diff
@@ -205,77 +205,95 @@ def preprocess(index: int, **kwargs) -> xr.Dataset:
 
     # new variables
     ds["drogue_status"] = (
-        ["traj", "obs"],
+        [gdp.GDP_DIMS["traj"], gdp.GDP_DIMS["obs"]],
         [gdp.drogue_presence(ds.drogue_lost_date.data, ds.time.data[0])],
     )
 
     # convert attributes to variable
     ds["location_type"] = (
-        ("traj"),
+        (gdp.GDP_DIMS["traj"]),
         [False if ds.get("location_type") == "Argos" else True],
     )  # 0 for Argos, 1 for GPS
-    ds["DeployingShip"] = (("traj"), gdp.cut_str(ds.DeployingShip, 20))
-    ds["DeploymentStatus"] = (("traj"), gdp.cut_str(ds.DeploymentStatus, 20))
-    ds["BuoyTypeManufacturer"] = (("traj"), gdp.cut_str(ds.BuoyTypeManufacturer, 20))
-    ds["BuoyTypeSensorArray"] = (("traj"), gdp.cut_str(ds.BuoyTypeSensorArray, 20))
+    ds["DeployingShip"] = ((gdp.GDP_DIMS["traj"]), gdp.cut_str(ds.DeployingShip, 20))
+    ds["DeploymentStatus"] = (
+        (gdp.GDP_DIMS["traj"]),
```

Comment:
But there is a problem here, as I said I was defining: `gdp.GDP_DIMS = {"rows": "raj," "obs": "traj"}`, so we can write adapters and put `(GDP.GDP_DIMS["rows"])` or `(GDP.GDP_DIMS["obs"])` as the dimension, and it would be renamed if needed. As we have it now, then the GDP would have "rows" as it's dimension.